forgevtt-module.js

```diff
@@ -97,6 +97,20 @@ class ForgeVTT {
     static isObjectEmpty(obj) {
         return !obj || typeof(obj) !== "object" || Object.keys(obj).length === 0;
     }
+
+    /**
+     * The global isNewerVersion will be removed in v14, so we need a utility function to alias to whichever is available.
+     * @param {string} version 
+     * @param {string} target 
+     */
+    static isNewerVersion(version, target) {
+        if (foundry?.utils) {
+            return foundry.utils.isNewerVersion(version, target);
+        } else {
+            return isNewerVersion(version, target);
+        }
+    }
```

Comment:
Shouldn't we avoid conditional chaining? Or is it just for the Forge repo?

Using `ForgeVTT.isNewerVersion(...)` might force to believe we are checking the Forge version, not the Foundry one... maybe we could rename it to `isFoundryNewerVersion`?
```suggestion
    static isFoundryNewerVersion(version, target) {
        if (foundry && foundry.utils) {
            return foundry.utils.isNewerVersion(version, target);
        } else {
            return isNewerVersion(version, target);
        }
    }
```

Reply:
That's only a restriction on the Forge repo. For Forge server side, it's due to Node 12. For Forge client side, it was because of the gulp Vue component parser not knowing how to parse that syntax and failing to process the file.

This JS runs directly in Foundry, i.e. the browser, so optional chaining is safe to use.